hw11/Lesson11.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+import inspect
+
+func_name = None
+
+
+def my_decorator_func(func):
+    def deco_func():
+        func()
```

Comment:
assign the result of the func call to some variable add return it after the last print to be able to decorate functions that return some value